### PR TITLE
Remove cleanup to avoid null canvas in unit tests

### PR DESCRIPTION
**Pull request by fyu**

### CODE REVIEW

**Comment by sinnyn** [comment]
@fyu These tests are fixed.

File: app/src/test/components/label2d_canvas_util.tsx [hunk 1]
```diff
@@ -111,14 +111,20 @@ export function keyUp (label2d: Label2dCanvas, key: string) {
...
+ * The canvas should move first to get _highlightedHandle changed, otherwise at
+ * the first stroke, the addVertex method would return true signals the polygon
+ * is completed.
  * @param label2d
  * @param points
  */
 export function drawPolygon (label2d: Label2dCanvas, points: number[][]) {
   const draw = () => {
     for (const p of points) {
+      mouseMove(label2d, p[0], p[1])
```
Comment (fyu) [inline app/src/test/components/label2d_canvas_util.tsx:124]: One mouse move should suffice.

**Comment by sinnyn** [comment]
Mouse move fixed @fyu

**Comment by fyu** [comment]
This is neat!

**Pull request merged**